docs/_docs/03-02-product-guide-commenter-features.md

```diff
@@ -0,0 +1,142 @@
+
+* This comment is offensive
+* This looks like an ad/marketing
+* I don’t agree with this comment
+* Other
+
+They can also include more information and this shows for moderators in the Flag Detail area on the comments in the moderation queues.
+
+Comments that are reported go to the Reported queue, with the exception of “I don’t agree with this comment”. This option is a useful way to let other readers vent their frustration, but since just disagreeing with something doesn’t mean it’s not suitable, we leave it be.
+
+
+## Reporting Usernames
+
+Usernames can also be reported by readers, if the username is inappropriate or offensive. They can choose one of the following reasons:
+
+* This username is offensive
+* I don't like this username
+* This user is impersonating
+* This looks like an ad/marketing
+
+Reported usernames go to the Reported Usernames queue which is located in the Community tab. If a username is rejected by a moderator, the commenter is prompted to change their username and they are suspended from commenting, replying or reacting to comments until they do so. They receive an email, and also a message at the top of their comment streams that let’s them know they’re suspended.
+
+If the commenter changes their username, it goes back to the Reported Usernames queue for approval. If the updated username is accepted by a moderator, the commenter is no longer suspended and continue interacting with the community. If the username is rejected, the commenter remains suspended until they change their username to something appropriate.
+
+Approved usernames that are reported do not show up in the Reported Usernames queues any longer, since they have been specifically OK’ed by a moderator.
+
+## Ignoring Users
+
+Commenters can ignore other commenters and essentially mute them entirely from the comment platform. Commenters can manage their ignored users list in their My Profile tab.
+
+## Featured Comments
+
+Moderators can feature comments that they want to highlight and recommend to their community. Featured comments show up on a separate tab, that is the default for the comment stream. Featured comments within the stream show a Featured badge.
+
+## Sorting the Stream
+
+Readers can sort the stream in 4 ways based on their viewing preferencet:
+
+* Oldest first
+* Newest first
+* Most respect first (or most liked, most loved, etc., depending on what reactions you use)
+* Most replied first
+
+We also make it easy to add more sorts via custom plugins.
+
+## Badges
+
+Badges differentiate users and comments on the stream. By default, Talk has two badges. 
+
+The Staff user badge that shows when a commenter has an Admin, Moderator, or Staff role. 
+
+The Featured comment badge shows when a comment has been featured.
+
+Another optional badge is the Subscriber badge (which is available as a [Recipe](/plugin-recipes/#recipe-subscriber). 
```

Comment:
Links should include the relative URL liquid template tag

Reply:
Done!